src/main/java/codeu/model/store/basic/MessageStore.java

```diff
@@ -130,14 +130,4 @@ public Message getMessageById(UUID id) {
   public int getMessageCount() {
     return messages.size();
   }
-
-  /** Returns message by Id */
-  public Message getMessageById(UUID id) {
-    for (Message message : messages) {
-      if (message.getId().equals(id)) { 
-        return message; 
-      }
-    }
-    return null; 
-  }
```

Comment:
Why did you delete this method?

Reply:
We had two of the same methods in this file